### ISSUE TITLE
missing OneNotePowershell.msi

### ISSUE THREAD

**Issue by Verayth** [sytone/OneNotePowerShellProvider#4]
Please provide a working OneNotePowershell.msi installer, and/or instructions on how to build it from source.  The link to the file in various blogs no longer works.  It'd be nice if it was posted as a release here on GitHub.

I was able to get a good compile in VS 2017 and DotNet4.8 after a bit of work, but I don't believe I know how to correctly register the snap-in with Powershell.  That's assuming I didn't break something.

If I just use an Import-Module on the DLL, I get a 'class not registered' error using the Open-OneNote command.  The OneNote provider shows up OK with Get-PSDrive.

``Open-OneNote : Retrieving the COM class factory for component with CLSID {D7FAC39E-7FF1-49AA-98CF-A1DDD316337E} failed due to the following error: 80040154 Class not
registered (Exception from HRESULT: 0x80040154 (REGDB_E_CLASSNOTREG)).``

**Comment by queks** [comment]
Did you manage to find the .msi file?

**Comment by sytone** [comment]
no, I searched through local files and did not find it. Will keep looking.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by queks** [comment]
Thanks. I tried looking for it from the original blog, but the OneDrive link the author points to is no longer valid.
